css/main.css

```diff
@@ -325,3 +325,125 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  margin: 120px 0 0 0;
+}
+
+.contact {
+  padding: 0 24px;
+}
+
+.description-contact {
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  margin: 0 0 73px 0;
+  color: #172b4d;
+}
+
+.contact-inputs {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+#full-name {
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  border-width: 1px;
+  width: 100%;
+}
+
+#email-address {
+  border-top: none;
+  border-right: none;
+  border-left: none;
+  border-width: 1px;
+  margin-top: 39px;
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.contact-inputs .small-validation {
+  background-color: #172b4d;
+  color: white;
+  padding: 5px 10px;
+}
+
+#text-area {
+  border: none;
+  background-color: #fbf8f7;
+  margin-top: 24px;
+  width: 80%;
+  height: 114px;
+}
+
+.container-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 24px 0 124px 0;
+}
+
+.button-submit {
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  height: 48px;
+  width: 131px;
+  background-color: #3700ff;
```

Comment:
You did a great job implementing the requested hover styles for the button! 💯 However, looks like you also changed the background color of the button to its normal state. Kindly remember to stick as much as possible to the Figma design as stated in the requirements, these include colors. Having a consistent design across your page will make it look very professional and attractive later on to recruiters. Please, fix it 🔧 

Your design:
![image](https://user-images.githubusercontent.com/77069011/197202711-3fc564fe-6166-42ae-aea2-34a07f32fa35.png)

Figma template:
![image](https://user-images.githubusercontent.com/77069011/197202802-bcabd1c3-e50b-4388-a252-51d60c451410.png)